test/player_test.rb

```diff
@@ -7,19 +7,17 @@ def setup
     @player = Noplayer.new
   end
 
-
   # Default name
 
   def test_default_name
-    assert_equal "Noplayer", @player.name
+    assert_equal 'Noplayer', @player.name
   end
 
   def test_default_name_should_remove_namespace
     flexmock(@player).should_receive(:class).and_return("Foo::Bar")
-    assert_equal "Bar", @player.name
+    assert_equal 'Bar', @player.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.